subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1317,12 +1317,9 @@ double GameLogicPluginPrivate::ScoreArtifact(const ArtifactType &_type,
     this->LogEvent(stream.str());
 
     this->duplicateReportCount++;
-    return 0.0;
+    return uniqueReport->second;
```

Comment:
the score change also gets added to the total score [here](https://github.com/osrf/subt/blob/d1b4a51e3813b51090ef83d4f97d5479f95d30ee/subt_ign/src/GameLogicPlugin.cc#L1204) so a duplicate report will end up scoring a point. I think that's not intended? Here's output from a quick test of reporting the same artifact twice which shows a total score of 2 instead of 1:

```
[Msg] SimTime[10 568000000] OnNewArtifact Msg=type: 5
pose {
  position {
    x: 191.80000000009215
    y: 20.00000000000043
    z: -15.069500004108624
  }
}

[Msg]   [Total]: 1
[Msg] Total score: 1
[Dbg] [VisibilityRfModel.cc:80] Range: 12.3695, Exp: 2.5, TX: 20, RX: -47.3088
[Dbg] [VisibilityRfModel.cc:80] Range: 12.3695, Exp: 2.5, TX: 20, RX: -47.3088
[Msg] SimTime[10 861000000] OnNewArtifact Msg=type: 5
pose {
  position {
    x: 191.80000000009215
    y: 20.00000000000043
    z: -15.069500004108624
  }
}

[Msg] This report has been received before
[Msg] Total score: 2
```

Reply:
Fixed in 24b37a188a2c81a62bc0384ddc73ea59b0b81e7c